CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Changed
+- fix: rscPrefix config without trailing slash #50
```

Comment:
do you have a CI to auto-generate the CHANGELOG?

Reply:
No, for now it's done manually. (In jotai, it's generated by GitHub. We may follow it eventually.)